ethcore/sync/src/blocks.rs

```diff
@@ -422,7 +420,7 @@ impl BlockCollection {
 					match self.blocks.get_mut(&h) {
 						Some(ref mut block) => {
 							trace!(target: "sync", "Got receipt {}", h);
-							block.receipts = Some(r.clone());
+							block.receipts = Some(r.to_vec());
```

Comment:
why not just move it and keep r: Bytes?

Reply:
it is set in a loop, so we need to copy the data in any case